config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-postsubmits.yaml

```diff
@@ -71,6 +107,42 @@ periodics:
     testgrid-tab-name: Kubernetes Master Driver Latest
     testgrid-alert-email: gke-managed-storage-alerts@google.com
     description: Kubernetes Integration tests for Kubernetes Master branch and Driver latest build
+- name: ci-gcp-compute-persistent-disk-csi-driver-latest-k8s-master-integration-canary-sidecars
+  interval: 4h
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220214-0aa8fe1d3a-master
+      args:
+      - "--repo=sigs.k8s.io/gcp-compute-persistent-disk-csi-driver"
+      - "--root=/go/src"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--clean"
+      - "--timeout=180" # Minutes (120m times out)
+      - "--scenario=execute"
+      - "--" # end bootstrap args, scenario args below
+      - "test/run-k8s-integration-ci.sh"
+      env:
+      - name: GCE_PD_OVERLAY_NAME
+        value: "prow-canary-sidecar"
```

Comment:
Overlay defined in pd csi driver [here](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/tree/2ca4c37e8172e104c30788cf204955ca0f4e3a4e/deploy/kubernetes/overlays/prow-canary-sidecar). This overlay uses the master PD CSI Driver and canary sidecars.